src/ziggurat/streams.clj

```diff
@@ -51,20 +67,26 @@
                            buffered-records-per-partition
                            commit-interval-ms
                            upgrade-from
-                           changelog-topic-replication-factor]}]
+                           changelog-topic-replication-factor
+                           default-key-serde
```

Comment:
can we rename this to something that makes more sense?
If we wish to keep it the same as kafka, then it should be `default-key-serde-class`.
I'd recommend to keep it to something else like `key-serde-class`.

Reply:
Hmm. Actually, `default-key-serde` will be consistent with `default.key.serde` config defined here: https://docs.confluent.io/current/streams/developer-guide/config-streams.html.